src/Util/Sniffs/TextStrings.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Utility functions for working with text string tokens.
+ *
+ * @author    Juliette Reinders Folmer <phpcs_nospam@adviesenzo.nl>
+ * @copyright 2019 Juliette Reinders Folmer. All rights reserved.
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Util\Sniffs;
+
+use PHP_CodeSniffer\Exceptions\RuntimeException;
+use PHP_CodeSniffer\Files\File;
+
+class TextStrings
```

Comment:
The name `Strings` to me would be very confusing as that could also be interpreted as being aimed at `T_STRING` tokens, which is why I made it more explicit by calling the class `TextStrings`.

Reply:
I think developers would understand that `strings` means the data type. If someone has a heredoc and they are looking for util methods, I think they'd look inside a `Strings` class. But I also think this class should be combined with `Orthography`, so these two conversations overlap substantially.

Reply:
Are there examples of "text" which is not a "string"?
Or "string" that is not "text"?

Reply:
"Strings" could be confused with `T_STRING`-related in context of tokens. That's a valid point.  

Also, generic names like `Strings` [are pain to use](https://www.tomasvotruba.cz/blog/2019/05/02/alias-as-a-code-smell/), since every package has the same short class name.

Reply:
Maybe make things really obvious StringUtil

Reply:
I'm partial to the `StringUtil` idea, or something like it. Clearly we're already in a `Util` namespace, but since there's ambiguity in the `String` name, I like the idea of making it more explicit. These are static methods, so the class name is sort of a function category anyway, rather than a data type. I think that `StringUtil` is a more discoverable category than `TextStrings`.

Reply:
I get what you're saying about `Strings` being a generic name, but I'm not a big fan of doubling up the use of `Util` in the fully qualified class name.

I could be convinced otherwise though. If we went this way, I think all the util classes would need `Util` appended for consistency.

One place I do like this is with the `TokenIs` class, which is in need of a better name. `TokenUtil` might be a good one.